### DISCUSSION TITLE
Issue Uploading Images through Atom

### DISCUSSION THREAD

**Discussion by annamorales5** [thegetty/quire#224] [Get Help] [answered]
Hello Quire community! I can't seem to figure out how to use yaml to add images (more specifically to change the background photo for the cover page of my local site). I saw that the Quire tutorial tells me to add the yaml to "image:" under my "cover.md" page, but when I change the image identification to the one I want to use, it does not change on my browser preview. I also added the image to my project folder on my desktop, and made sure the file is in .jpeg format, but no luck. Does anyone know how I can import photos through my text editor?

**Comment by Erin-Cecele** [comment]
Hi @annamorales5. Make sure that the jpeg is saved in the correct place. You want to save it to the `img` folder that is found in your `static` folder. I also recommend double-checking that you added your image to the same project folder that you are previewing. You mention that it is saved on your Desktop but Quire projects are often saved in the home directory by default. (You can find the directory by opening your Finder and typing Command-Shift-H.)

**Comment by annamorales5** [comment]
Hi Erin,

Thanks for this suggestion. I checked and the .jpeg file is placed in my "my-project" folder within my home directory. As seen in this screenshot, I entered the file name (tried both with and without the .jpeg at the end), but the local preview is not changing the cover page. Do you have any other ideas?

Thanks,
Anna

-------------------------------------------------------------------------------------------------------------------------------------------------
Anna Morales
Photographer & Visual Anthropologist
Cell: 267-901-7001
Columbia, SC & Philadelphia, PA
https://avisualanthropologist.com/





________________________________

**Reply by Erin-Cecele** [reply]
Hi @annamorales5. Can you send a screenshot of the structure of your folders in Atom so I can see where the image is saved, and then a screenshot of the cover.md file? Thank you!

**Comment by annamorales5** [comment, marked as answer]
Erin,

As I was sending over the information, I realized that the issue is I was typing in the file as "jpeg" rather than "jpg". I changed this and it solved the issue! Thanks for your help.

Best,
Anna

-------------------------------------------------------------------------------------------------------------------------------------------------
Anna Morales
Photographer & Visual Anthropologist
Cell: 267-901-7001
Columbia, SC & Philadelphia, PA
https://avisualanthropologist.com/





________________________________

**Reply by Erin-Cecele** [reply]
That's wonderful news @annamorales5. Let us know if anything else comes up!

**Comment by annamorales5** [comment]
Thanks, 🙂 Have a great night.

Best,
Anna

-------------------------------------------------------------------------------------------------------------------------------------------------
Anna Morales
Photographer & Visual Anthropologist
Cell: 267-901-7001
Columbia, SC & Philadelphia, PA
https://avisualanthropologist.com/





________________________________
